docker/Dockerfile.release

```diff
@@ -0,0 +1,76 @@
+# The MIT License (MIT)
```

Comment:
Maybe it can be split into (and avoid problems with having to update the code in two places):
- Dockerfile.base
- Dockerfile.devel = Dockerfile.base + gdb + CMAKE_BUILD_TYPE=Debug
- Dockerfile.release =Dockerfile.base + cmake

Or it would be too much for the user?

Reply:
IMHO, the redundancy right now is not high and makes the code easier to read and understand, so I'd personally leave it as it is

Reply:
Sure